Sources/Sentry/SentryHttpStatusCodeRange.h

```diff
@@ -0,0 +1,20 @@
+#import "SentryDefines.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(HttpStatusCodeRange)
+@interface SentryHttpStatusCodeRange : NSObject
```

Comment:
`m`: Please move this class into `/Sources/Sentry/includes`. I'm going to add a GH action and a pre-commit hook for this, as I quite often make this comment 😁